tiller/templates/serviceaccount.yaml

```diff
@@ -12,8 +12,28 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 ---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups:
+  - ""
+  - "batch"
+  - "extensions"
+  - "apps"
+  - "autoscaling"
+  - "aws.chatwork"
```

Comment:
@cw-sakamoto 
Tiler Chart should not depend on something.

https://github.com/helm/helm/blob/master/docs/rbac.md#example-deploy-tiller-in-a-namespace-restricted-to-deploying-resources-only-in-that-namespace
You should default to example and others inject fron `values.yaml`.

Reply:
That's right...I modified at https://github.com/chatwork/charts/pull/44/commits/b67328b01e244707e3ccd3f5599dcbee5ade7e5d